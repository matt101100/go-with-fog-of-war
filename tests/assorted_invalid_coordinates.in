place A20
place S20
place aaa
place Z1
place A
place 1
place 1A
place 1a
place a1
place A.
place 1.
place A232589473593845
who
term
