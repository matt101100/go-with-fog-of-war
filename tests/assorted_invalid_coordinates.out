Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
B
